src/openrct2/platform/Platform2.h

```diff
@@ -44,7 +44,7 @@ namespace Platform
 #ifdef _WIN32
     bool IsOSVersionAtLeast(uint32_t major, uint32_t minor, uint32_t build);
 #endif
-
+    bool is_running_in_wine();
```

Comment:
This should follow the style of the file (which is actually our coding guideline
```suggestion
    bool IsRunningInWine();
```